### PR TITLE
Password Verification

### PR DESCRIPTION
**Pull request by mread1208**
Adding new Regex to check for lower, capital, number, special char, and at least 8 chars.  Creating a binding check for whether the confirmPassword field matches the default password field.

### CODE REVIEW

**Comment by mread1208** [comment]
Update for issue #30

File: src/app/pages/register/register.component.ts [hunk 1]
```diff
@@ -57,4 +59,10 @@ export class RegisterComponent implements OnInit {
       this.toastrService.error('An error occurred while registering', 'Error');
     });
   }
+
+  private validateAreEqual(fieldControl: FormControl) {
```
Comment (bartdebever) [inline src/app/pages/register/register.component.ts]: Please indicate the return value with `:void` This is super minor and I'm testing it right now, if it works I'll let this slide

**Comment by bartdebever** [comment]
Works perfectly, I'll fix the tiny issue with the return value later, thank you!

File: src/app/pages/register/register.component.ts (hunk 1 above)
Comment (mread1208) [reply src/app/pages/register/register.component.ts]: Ah, thanks good catch, apologies.  Thanks for merging this in.

**Pull request merged**